estuary_updater/handlers/base.py

```diff
@@ -89,42 +89,53 @@ def is_container_build(self, build_info):
         else:
             return False
 
-    def get_or_create_build(self, identifier, original_nvr=None, force_container_label=False):
+    def get_or_create_build(self, identifier, build_info=None, original_nvr=None,
+                            force_container_label=False):
         """
         Get a Koji build from Neo4j, or create it if it does not exist in Neo4j.
 
         :param str/int identifier: an NVR (str) or build ID (int)
+        :kwarg dict build_info: all info needed to create a build, else get it from Koji
```

Comment:
I'd say since it's unlikely the messages will be fixed to include the proper build info in the near future, I'd say you can just remove this option and call the API every time. We will reimplement something like this when it does get fixed,